lib/commands/connect/shared.js

```diff
@@ -109,77 +110,69 @@ let request = exports.request = function(token, method, path, data){
   return promise;
 };
 
-let withUserConnections = exports.withUserConnections = function(token, appName, flags, allowNone, heroku){
+let withUserConnections = exports.withUserConnections = co.wrap(function* (token, appName, flags, allowNone, heroku){
```

Comment:
I don't know if I actually need `co.wrap` here. I know I need the asterisk in order to make it a generator, but the code seems to work just fine with and without `co.wrap`. Apparently `co.wrap` turns a generator into a promise, and I don't know if that's actually necessary here or not.